source/code/source/playfab/PlayFabEventApi.cpp

```diff
@@ -26,7 +26,7 @@ namespace PlayFab
         this->eventRouter->RouteEvent(eventRequest);
     }
 
-    void PlayFabEventAPI::EmitEvent(std::unique_ptr<const IPlayFabEvent> event, std::function<void(std::shared_ptr<const IPlayFabEvent>, std::shared_ptr<const IPlayFabEmitEventResponse>)> callback) const
+    void PlayFabEventAPI::EmitEvent(std::unique_ptr<const IPlayFabEvent> event, std::function<void(const std::shared_ptr<const IPlayFabEvent>&, const std::shared_ptr<const IPlayFabEmitEventResponse>&)> callback) const
```

Comment:
disagree

Reply:
This derives from my reply above. This is a public facing callback function whose signature cannot change without breaking users.